.github/ISSUE_TEMPLATE.md

```diff
@@ -39,6 +39,7 @@ What needs to be done to address this issue? Ideally, provide a pull request wit
 ### Additional Information
 
 * React Native version: [FILL THIS OUT: Be specific, filling out "latest" here is not enough.]
+* Node/npm version: [FILL THIS OUT: `node -v` and `npm -v`]
```

Comment:
Added yarn next to npm, gave node its own line. Also reordered and added hint about `react-native info` to coincide with https://github.com/facebook/react-native/pull/14428